pkg/cephmgr/rgw/admin.go

```diff
@@ -22,8 +22,15 @@ import (
 	"github.com/rook/rook/pkg/clusterd"
 )
 
+type AdminCommandRunner interface {
+	RunAdminCommand(context *clusterd.Context, command, subcommand string, args ...string) (string, error)
```

Comment:
why do we need a new interface to mock these commands? We already have the interface for shelling out commands, which will happen when it gets to the line where context.ProcMan.RunWithOutput() is called. That's the level where we've been mocking throughout.